examples/ChakraUI/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "@freshheads/cookie-guard-example-chakra-ui",
```

Comment:
Is het idee hier dat je deze package.json niet in deze library build maar puur als voorbeeld gebruikt ?
Of kun je dit met een script in de main package.json laten builden en daar een server voor laten draaien?

Reply:
Ja inderdaad puur als voorbeeld, door dit zo op te zetten met eigen node_modules is het makkelijker om de package te developen. Je krijgt gewoon je imports en type checking etc. En om de example te gebruiken kun kijken wat er in de package.json staat. (Packages die buiten deze example niet nodig zijn, zoals chakra-ui)

Reply:
Moeten alleen straks nog even checken of dat goed gaat, dat bijvoorbeeld niet chakra-ui als dependency van de cookie-guard komt.

Reply:
Ja dat klopt, deze example kan je niet standaard builden vanuit de package